express/package.json

```diff
@@ -9,7 +9,7 @@
     "dev": "COGNITO_CLIENT=${COGNITO_CLIENT:-StubCognitoClient} LAMBDA_FACADE=${LAMBDA_FACADE:-StubLambdaFacade} nodemon --config nodemon.json -r 'dotenv/config' src/app.ts",
     "buildts": "tsc -p .",
     "buildsass": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js assets/css/app.scss dist/assets/app.css",
-    "buildjs": "uglifyjs --verbose node_modules/govuk-frontend/govuk/all.js -o dist/assets/govuk-all.js && uglifyjs --verbose assets/javascripts/cookies.js -o dist/assets/cookies.js && cp -v -r assets/javascripts/show-password.js ./dist/assets",
+    "buildjs": "uglifyjs --verbose node_modules/govuk-frontend/govuk/all.js -o dist/assets/govuk-all.js && uglifyjs --verbose assets/javascripts/cookies.js -o dist/assets/cookies.js && cp -v -r assets/javascripts/show-password.js ./dist/assets && cp -v -r assets/javascripts/show-password-two-fields.js ./dist/assets",
```

Comment:
Can we copy all the files from the `express/assets/javascripts` directory instead of having to add a copy command each time we add a new js file?

Reply:
No at the moment we can not. Some of the files are optimized and some are written in a way that can not be optimized. That's why some of the files are optimized and some are copied, and we can not just move them all at once.